releasenotes/notes/pending-deprecate-mqwv-matrix-setter-1cd9e2dfb68d1a85.yaml

```diff
@@ -1,7 +1,6 @@
 ---
 deprecations:
   - |
-    The M, Q, W, V matrix setters from
+    The M, Q, W, V matrix setters and M, Q, W, V matrix standard deviation setters from
     :class:`~qiskit_nature.second_q.algorithms.excited_state_solvers.qeom.QEOMResult`
```

Comment:
```suggestion
    :class:`~qiskit_nature.second_q.algorithms.excited_state_solvers.qeom.QEOMResult`
```

Reply:
I think that typo prevents it coming out as a link

Reply:
oops sry about that